[Issue #155] feat: Display networks

chedieck opened the pull request:
Description:
Shows the available networks on http://localhost:3000/networks

Test plan:
http://localhost:3000/networks should display:
![image](https://user-images.githubusercontent.com/21281174/185163262-85f33833-dcb7-4263-a660-cb688a429830.png)

Klakurka reviewed (changes requested):
Can we add a config option (default off) to show test networks?

chedieck commented:
> Can we add a config option (default off) to show test networks?

Addressed, added a `SHOW_TEST_NETWORKS` env var which defaults to  `false` if not present.
